test/parallel/forall/vass/par-loops/2-enclosing-par-loops-error-4.chpl

```diff
@@ -0,0 +1,41 @@
+/*
+This tests the situation where a reduce intent and/or in intent
+occurs in a forall loop, whose the parallel iterator contains a yield
+that is in 2 parallel loops.
+
+Specifically, in a parallel for loop that's in another parallel for loop.
+
+When this is no longer an error, test for absence of data races, like
+  parallel/forall/vass/reduce-with-forall-in-par-iter
+  parallel/forall/vass/reduce-fatter-with-forall-in-par-iter
+*/
+
+iter myiter() {
+  yield 555;
+}
+
+iter myiter(param tag) {
+  coforall ooo in 1..3 {
+    yield 666;
+  }
+}
+
+iter pariter() {
+  yield 333;
+}
+
+iter pariter(param tag) {
```

Comment:
Why doesn't this check tag== iterKind.standalone?

Reply:
To explain my thinking:  the check tag== iterKind.standalone is not needed in this simple test. In the sense that it should not affect the test outcome or the thing being tested. I omitted this check to shorted the code.

Reply:
It seems to me that we should prefer easy to understand test cases over shorter test cases. They are not the same thing. Standalone iterators normally include the where clause.